low-level/files/opt/seagate/sspl/setup/resource_map/server.py

```diff
@@ -73,7 +77,11 @@ def get_health_info(self, rpath):
                 fru, _ = self.get_node_details(node)
                 if self.server_frus.get(fru):
                     fru_found = True
-                    info = self.server_frus[fru]()
+                    try:
+                        info = self.server_frus[fru]()
+                    except:
+                        # TODO: Log the exception
+                        info = None
```

Comment:
Added exception handling here to avoid breaking any other ongoing FRUs call.
cc: @sumedhak27